server/datastore/mysql/labels.go

```diff
@@ -174,10 +174,10 @@ func batchHostnames(hostnames []string) [][]string {
 	// https://github.com/golang/go/wiki/SliceTricks#batching-with-minimal-allocation
 	//
 	// WARNING: This is used in ApplyLabelSpecsWithAuthor and the batch sizes have to be small
-	// enough to allow for three copies each hostname list in the query. The batch size is 20_000
+	// enough to allow for three copies each hostname list in the query. The batch size is 15_000
 	// because 60_001 binding arguments is less than the maximum of 65,535.
 
-	const batchSize = 20_000 // Large, but well under the undocumented limit
+	const batchSize = 15_000 // Large, but well under the undocumented limit
```

Comment:
Because we're adding another arg binding to the SQL query above, we now have 4 bindings so to keep it at 60k total per patch we need to drop the batch size down.